app/models/alchemy/language.rb

```diff
@@ -26,6 +26,7 @@ class Language < ActiveRecord::Base
     validates_presence_of :language_code
     validates_presence_of :page_layout
     validates_presence_of :frontpage_name
+    validates_presence_of :locale, if: -> { Alchemy::I18n.app_locales.present? }
```

Comment:
Rails/Validation: Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.